.idea/checkstyle-idea.xml

```diff
@@ -3,11 +3,13 @@
   <component name="CheckStyle-IDEA">
     <option name="configuration">
       <map>
+        <entry key="active-configuration" value="LOCAL_FILE:D:/Student Data/Documents/checkstyle.xml:P2pStyle" />
         <entry key="checkstyle-version" value="8.24" />
         <entry key="copy-libs" value="true" />
         <entry key="location-0" value="BUNDLED:(bundled):Sun Checks" />
         <entry key="location-1" value="BUNDLED:(bundled):Google Checks" />
         <entry key="location-2" value="PROJECT_RELATIVE:$PROJECT_DIR$/.idea/codeStyles/checkstyle.xml:CheckStyle" />
+        <entry key="location-3" value="LOCAL_FILE:D:/Student Data/Documents/checkstyle.xml:P2pStyle" />
```

Comment:
change to a relative location if possible. this will break on other people's machines

Reply:
I noticed that the checkstyle file included with the repository was not the same as the one on the wiki.

Reply:
I must be going crazy. The file in the repo is the same.